src/test/java/it/EndpointIT.java

```diff
@@ -23,7 +23,7 @@
 import javax.ws.rs.client.WebTarget;
 import javax.ws.rs.core.Response;
 
-public class EndpointTest {
+public class EndpointIT {
```

Comment:
In this project, since EndpointIT doesn't actually contain any @Test annotations, it can be named anything like EndpointHelper.   Then TestApplication should be renamed ApplicationIT.java